WinUIGallery/Samples/ControlPages/AppWindowPage.xaml

```diff
@@ -86,6 +86,15 @@
                             SpinButtonPlacementMode="Inline"
                             Value="50" />
                     </Grid>
+
+                    <TextBlock FontWeight="SemiBold"
+                               Text="TitleBar theme" />
+                    <ComboBox x:Name="TitleBarPreferredTheme" SelectedIndex="0" HorizontalAlignment="Stretch">
```

Comment:
It is also better to bind these enum items with x:bind instead of hardcoding them to protect them from any changes.